src/YieldDistributor.sol

```diff
@@ -97,8 +107,12 @@ contract YieldDistributor is IYieldDistributor, OwnableUpgradeable {
      * @return uint256 The voting power of the user
      */
     function getCurrentVotingPower(address _account) public view returns (uint256) {
-        return
-            this.getVotingPowerForPeriod(BREAD, lastClaimedBlockNumber - cycleLength, lastClaimedBlockNumber, _account);
+        return this.getVotingPowerForPeriod(
+            BREAD, lastClaimedBlockNumber - cycleLength, lastClaimedBlockNumber, _account
+        )
+            + this.getVotingPowerForPeriod(
```

Comment:
Calling this twice is pretty inefficient. Wonder if it's possible to easily refactor `getVotingPowerForPeriod` function to calculate both values in one pass.

Reply:
I'm not sure there's a better way to do it... they will likely have different checkpoints and you have to iterate over all of them in the range for both tokens